specification/extendedlocation/resource-manager/Microsoft.ExtendedLocation/preview/2021-08-31-preview/customlocations.json

```diff
@@ -0,0 +1,1196 @@
+          "type": "array",
+          "items": {
+            "type": "string"
+          }
+        }
+      }
+    },
+    "patchableCustomLocations": {
+      "description": "The Custom Locations patchable resource definition.",
+      "type": "object",
+      "x-ms-azure-resource": true,
+      "x-ms-client-flatten": true,
+      "properties": {
+        "identity": {
+          "$ref": "#/definitions/Identity",
+          "description": "Identity for the resource."
+        },
+        "properties": {
+          "$ref": "#/definitions/customLocationProperties",
+          "description": "The Custom Locations patchable properties.",
+          "x-ms-client-flatten": true
+        },
+        "tags": {
+          "description": "Resource tags",
+          "type": "object",
+          "additionalProperties": {
+            "type": "string"
+          }
+        }
+      }
+    },
+    "patchableResourceSyncRule": {
+      "description": "The Resource Sync Rules patchable resource definition.",
+      "type": "object",
+      "x-ms-azure-resource": true,
+      "x-ms-client-flatten": true,
+      "properties": {
+        "properties": {
+          "$ref": "#/definitions/resourceSyncRuleProperties",
+          "description": "The Resource Sync Rules patchable properties.",
+          "x-ms-client-flatten": true
+        },
+        "tags": {
+          "description": "Resource tags",
+          "type": "object",
+          "additionalProperties": {
+            "type": "string"
+          }
+        }
+      }
+    },
+    "resourceSyncRule": {
+      "description": "Resource Sync Rules definition.",
+      "type": "object",
+      "properties": {
+        "properties": {
+          "$ref": "#/definitions/resourceSyncRuleProperties",
+          "description": "The set of properties specific to a Resource Sync Rule",
+          "x-ms-client-flatten": true
+        },
+        "systemData": {
+          "$ref": "../../../../../common-types/resource-management/v2/types.json#/definitions/systemData",
+          "description": "Metadata pertaining to creation and last modification of the resource",
+          "readOnly": true
+        }
+      },
+      "allOf": [
+        {
+          "$ref": "../../../../../common-types/resource-management/v2/types.json#/definitions/TrackedResource"
+        }
+      ]
+    },
+    "resourceSyncRuleListResult": {
+      "description": "The List Resource Sync Rules operation response.",
+      "type": "object",
+      "properties": {
+        "nextLink": {
+          "description": "The URL to use for getting the next set of results.",
+          "type": "string",
+          "readOnly": true
+        },
+        "value": {
+          "description": "The list of Resource Sync Rules.",
+          "type": "array",
+          "readOnly": true,
+          "items": {
+            "$ref": "#/definitions/resourceSyncRule"
+          }
+        }
+      }
+    },
+    "resourceSyncRuleProperties": {
```

Comment:
what are these used for?  I.e. when I create one what resources does it evaluate for "matches" and what happens with those resources?

Reply:
I updated the description in the swagger with more details.